pkg/collector/py/tests/testaggregator.py

```diff
@@ -22,12 +22,7 @@ def check(self, instance):
         self.gauge("testmetric", 0, tags=None)
         self.gauge("testmetricnonevalue", None) # metrics with None values should be ignored by AgentCheck
         self.gauge("testmetricstringvalue", "2") # string values should be cast to floats
-        try:
-            self.gauge("testmetricstringvalue", "notcastabletofloat") # values not castable to floats should raise an exception
-        except ValueError:
-            pass
-        else:
-            raise Exception("Expected gauge to raise ValueError")
+        self.gauge("testmetricstringvalue", "notcastabletofloat") # values not castable to floats should raise an exception
```

Comment:
`# values not castable to floats should raise an exception` is no longer true right ?

Reply:
Good catch, updated